samples/meeting-av-filters/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meeting-av-filters-csharp",
+    "source": "officeDev",
+    "title": "Meeting AV Filters",
+    "shortDescription": "This sample illustrates the usage of AV filters for Teams meeting.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meeting-av-filters/csharp",
+    "longDescription": [
+      "This sample demonstrates use of AV filters for teams meetings by processing the teams meeting video frames."
```

Comment:
This sample demonstrates use of AV filters for Teams meetings by processing the Teams meeting video frames.